source/ui/IDEAS/IDEASXASScanConfigurationView.cpp

```diff
@@ -398,3 +420,17 @@ void IDEASXASScanConfigurationView::onFluorescenceChoiceChanged(int id)
 {
 	configuration_->setFluorescenceDetector(id);
 }
+
+void IDEASXASScanConfigurationView::onPeakingTimeChanged(double value)
```

Comment:
In Issue1159 that last request was for the any changes in peaking time made in the detector view were also shown in the scan configuration view. I have the scan configuration view listen for peaking changes made in the detector.